perf/small_alloc_perf.c

```diff
@@ -181,103 +253,206 @@ print_json_test_finish(const char * finish)
 }
 
 static void
-print_json_test_result(double time)
+print_json_test_result(const unsigned long long mrps)
+{
+	size_t x = snprintf(json_output + pos, length,
+			    "                \"%llu\"\n", mrps);
+	length -= x;
+	pos += x;
+}
+
+static void
+print_json_test_result_double(const double mrps)
 {
 	size_t x = snprintf(json_output + pos, length,
-			    "                \"%.3f\"\n", time);
+			    "                \"%.3f\"\n", mrps);
 	length -= x;
 	pos += x;
 }
 
 static void
-small_alloc_basic(unsigned int slab_size)
+small_alloc_basic(const unsigned int slab_size)
 {
 	struct timespec tm1, tm2;
-	if(human) {
-		fprintf(stderr, "|              SMALL RANDOM "
-			"ALLOCATION RESULT TABLE                  |\n");
-		fprintf(stderr, "|___________________________________"
-			"_________________________________|\n");
-		fprintf(stderr, "|           alloc_factor          "
-			" |   	         time, ms            |\n");
-		fprintf(stderr, "|__________________________________|"
-			"_________________________________|\n");
-	} else {
-		print_json_test_header("random");
+	if (!no_basic_random_test) {
+		/*
+		 * Memory allocation/deallocation performance test for random size objects.
+		 */
+		if(human) {
+			fprintf(stderr, "|              SMALL RANDOM "
+				"ALLOCATION RESULT TABLE                  |\n");
```

Comment:
These prints repeat the same pattern in different places. Mb it is worth extracting it to the function?